vector/src/main/res/values/strings.xml

```diff
@@ -764,6 +764,10 @@
     <string name="search_members_hint">Filter room members</string>
     <string name="search_banned_user_hint">Filter banned users</string>
     <string name="search_no_results">No results</string>
+    <plurals name="search_space_parents">
+        <item quantity="one">%1$s and %2$s</item>
+        <item quantity="other">%1$s and %2$d others</item>
+    </plurals>
```

Comment:
This is not the correct way to handle plurals, it can lead to errors with translations.
Also the resource name is a bit strange to me
You must have something like:
```xml
    <strings name="search_space_two parents">%1$s and %2$s</string>
    <plurals name="search_space_parents">
        <item quantity="one">%1$s and %2$d other</item>
        <item quantity="other">%1$s and %2$d others</item>
    </plurals>
```

This is managed the same way for `room_displayname_two_members` if you want to have a look on it.